Makefile

```diff
@@ -0,0 +1,4 @@
+cs-fix:
+	./vendor/bin/phpcbf
+cs-check:
+	./vendor/bin/phpcs
```

Comment:
Try to add an extra empty line here :smile:

Reply:
And if you want, add this alternative to the main README, it could be faster than executing the individual command. In fact, I would add also the php-lint command (See GithHub actions to understand what I mean)